docs/dashboards/entity-dashboard.asciidoc

```diff
@@ -50,24 +80,41 @@ Interact with the table to filter data, view more details, and take action:
 
 For more information about host risk scores, refer to <<entity-risk-scoring>>. 
 
-[[entity-user-risk-scores]]
+[[entity-entities]]
 [float]
-== User Risk Scores
+== Entities
 
-Displays user risk score data for your environment, including the total number of users, and the five most recently recorded user risk scores, with their associated user names, risk data, and number of detection alerts. Like host risk scores, user risk scores are calculated using a weighted sum on a scale of 0 (lowest) to 100 (highest). 
+preview::[]
+
+.Requirements
+[sidebar]
+-- 
+To display the **Entities** section, you must <<enable-entity-store,enable the entity store>>.
+-- 
+
+The **Entities** section provides a centralized view of all hosts and users in your environment. It displays entities stored in the <<entity-store, entity store>>, which:
+
+* Have been observed by {elastic-sec}
+* Have an asset criticality assignment
+* Have been added to {elastic-sec} through an integration, such Active Directory or Okta
+
+NOTE: The **Entities** table only shows a subset of the attributes stored in the entity store. You can query the `.entities.v1.latest.security_user_<space-id>` and `.entities.v1.latest.security_host_<space-id>` indices to see all the attributes stored in the entity store.
 
 [role="screenshot"]
-image::images/user-score-data.png[User risk table]
+image::images/entities-section.png[Entities section] 
 
-Interact with the table to filter data, view more details, and take action:
+The stored entity data appears in the **Entities** section based on the following timelines:
 
-* Select the *User risk level* menu to filter the chart by the selected level. 
-* Click a user name link to open the user details flyout. 
-* Hover over a user name link to display inline actions: *Add to timeline*, which adds the selected value to Timeline, and *Copy to Clipboard*, which copies the user name value for you to paste later. 
-* Click *View all* in the upper-right to display all user risk information on the Users page. 
-* Click the number link in the *Alerts* column to view the alerts on the Alerts page. Hover over the number and select *Investigate in timeline* (image:images/timeline-button-osquery.png[Investigate in timeline icon,20,20]) to launch Timeline with a query that includes the associated user name value.
+* Observed events from the {elastic-sec} default data view are processed in near real-time.
```

Comment:
Please wordsmith this suggestion however it can best be conveyed!

```suggestion
* When first enabling the entity store, only data stored within the last 24 hours will be processed. Once enabled, entity data will be continually processed from then on.
* Observed events from the {elastic-sec} default data view are processed in near real-time.
```